.github/workflows/mutation-performance-baseline.yml

```
@@ -0,0 +1,197 @@
+          persist-credentials: false
+      - name: Prove frozen subject
+        shell: bash
+        run: |
+          set -euo pipefail
+          [[ "$SUBJECT_SHA" =~ ^[0-9a-f]{40}$ ]]
+          test "$(git rev-parse HEAD)" = "$SUBJECT_SHA"
+      - name: Set up Node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: .nvmrc
+          package-manager-cache: false
```

actionlint (1.7.12)

[error] input "package-manager-cache" is not defined in action "actions/setup-node@v4". available inputs are "always-auth", "architecture", "cache", "cache-dependency-path", "check-latest", "mirror", "mirror-token", "node-version", "node-version-file", "registry-url", "scope", "token"

(action)